src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -421,6 +422,26 @@ export type GeneticEntity = {
     geneticEntityData: Gene | Geneset;
 };
 
+// this can be adapted to consume new profile membership data
+export function allowExpressionProfiles(studies: CancerStudy[]) {
+    if (
+        /\.cbioportal\.org$|\.mskcc\.org$/.test(
+            getBrowserWindow().location.hostname
+        )
+    ) {
+        // if there is only one study
+        // or ALL studies are part of pan_can_atlas and thus have normalized data across expression profiles
+        if (
+            studies.length === 1 ||
+            _.every(studies, s => /pan_can_atlas/i.test(s.studyId))
+        ) {
+            return true;
+        }
```

Comment:
this is missing an `else` condition, we may end up implicitly returning `undefined` for some cases. How about we just return the evaluated condition?

```suggestion
        return (
            studies.length === 1 ||
            _.every(studies, s => /pan_can_atlas/i.test(s.studyId))
        );
```